fe/fe-core/src/main/java/com/starrocks/mysql/NegotiateState.java

```diff
@@ -20,6 +20,7 @@ public enum NegotiateState {
     READ_AUTH_SWITCH_PKG_FAILED("read auth switch package failed"),
     ENABLE_SSL_FAILED("enable ssl failed"),
     READ_SSL_AUTH_PKG_FAILED("read ssl auth package failed"),
+    SERVER_REJECTED_INSECURE_CONNECTION("server rejected the insecure connection"),
```

Comment:
nit: maybe align the naming with `Config` class. also maybe remove `server` for consistency with other state enum  msg.
```
 REQUIRE_SECURE_TRANSPORT(" insecure connection rejected"),
```